src/TimeService.Business/Commands/Import/ImportStatCommand.cs

```diff
@@ -124,22 +124,24 @@ private List<ProjectUserData> GetProjectsUsers(List<Guid> usersIds, List<string>
       return null;
     }
 
-    private List<Guid> FindDepartmentUsers(Guid departmentId, DateTime filterbyEntryDate, List<string> errors)
+    private async Task<List<Guid>> FindDepartmentUsersAsync(Guid departmentId, DateTime filterbyEntryDate, List<string> errors)
     {
       string messageError = "Cannot get department users info. Please, try again later.";
       const string logError = "Cannot get users of department with id: '{id}'.";
 
       try
       {
-        IOperationResult<IGetDepartmentUsersResponse> result = _rcGetDepartmentUsers.GetResponse<IOperationResult<IGetDepartmentUsersResponse>>(
-            IGetDepartmentUsersRequest.CreateObj(departmentId, ByEntryDate: filterbyEntryDate)).Result.Message;
+        Response<IOperationResult<IGetDepartmentUsersResponse>> result =
+          await _rcGetDepartmentUsers.GetResponse<IOperationResult<IGetDepartmentUsersResponse>>(
+            IGetDepartmentUsersRequest.CreateObj
+            (departmentId, ByEntryDate: filterbyEntryDate));
```

Comment:
make like this:
IGetDepartmentUsersRequest.CreateObj(
..departmentId, ByEntryDate: filterbyEntryDate)) (spaces were deleted so i write .)